### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, stat '/home/quincy/maestro/ma-core/src/main/ja...

### ISSUE THREAD

**Issue by SlimeQ** [bradleyboy/tidy-tabs-atom#6]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.7
**System**: ubuntu 14.04
**Thrown From**: [tidy-tabs](https://github.com/bradleyboy/tidy-tabs-atom) package, v0.2.0

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, stat '/home/quincy/maestro/ma-core/src/main/java/com/mobileagility/core/payload/flex/FlexTrendingList.java'

```
At fs.js:846

Error: ENOENT: no such file or directory, stat '/home/quincy/maestro/ma-core/src/main/java/com/mobileagility/core/payload/flex/FlexTrendingList.java'
    at Error (native)
    at fs.statSync (fs.js:846:18)
    at Object.fs.statSync (ATOM_SHELL_ASAR.js:206:16)
    at Object.module.exports.TidyTabs.isCandidate (/home/quincy/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:81:17)
    at Object.<anonymous> (/home/quincy/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:95:20)
    at Object.module.exports.TidyTabs.getItemsToClose (/home/quincy/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:100:10)
    at Object.module.exports.TidyTabs.tidy (/home/quincy/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:63:19)
    at atom-workspace.disposables.add.atom.commands.add.tidy-tabs:remove-stale-tabs (/home/quincy/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:43:26)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
```

### Commands

```
     -3:28.4.0 core:move-up (atom-text-editor.editor.is-focused)
  2x -3:28 core:move-right (atom-text-editor.editor.is-focused)
  2x -3:27.1.0 editor:select-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -3:24.8.0 core:move-right (atom-text-editor.editor.is-focused)
  5x -3:24.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -3:22.2.0 core:save (atom-text-editor.editor.is-focused)
     -2:13 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
     -2:09.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:06.6.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
     -2:01.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:33.9.0 core:copy (atom-text-editor.editor.is-focused)
     -1:32.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -1:32.2.0 core:paste (atom-text-editor.editor.is-focused)
  3x -1:25 core:backspace (atom-text-editor.editor.is-focused)
     -1:18.4.0 core:save (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -0:08.4.0 tidy-tabs:remove-stale-tabs (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "markdown-preview"
    ]
  },
  "tidy-tabs": {}
}
```

### Installed Packages

```coffee
# User
Remote-FTP, v0.7.2
Sublime-Style-Column-Selection, v1.3.0
browser-plus, v0.0.34
editor-stats, v0.17.0
git-plus, v5.4.7
java-importer, v1.4.0
language-ejs, v0.2.0
language-gradle, v0.0.3
language-salesforce, v0.6.0
markdown-preview-plus, v1.7.0
project-manager, v1.15.11
set-syntax, v0.3.0
tabs-to-spaces, v0.11.1
tidy-tabs, v0.2.0

# Dev
No dev packages
```

**Comment by SlimeQ** [comment]
this happened because i renamed a file and left it open, then tried to close stale tabs.

**Comment by anilanar** [comment]
Duplicate of #4.

**Comment by bradleyboy** [comment]
Should be resolved in 0.3.0. Let me know if you still have problems.
